Postman/index.js

```diff
@@ -1,27 +1,21 @@
-"use strict";
+require('babel/register');
 
-var Cronjob = require('cron').CronJob;
-var moment = require('moment-timezone');
+const Cronjob = require('cron').CronJob;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).